dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/AbstractCrudController.java

```diff
@@ -370,18 +371,20 @@ public void partialUpdateObject(
         // we don't allow changing UIDs
         ((BaseIdentifiableObject) patchedObject).setUid( persistedObject.getUid() );
 
-        MetadataImportParams params = importService.getParamsFromMap( contextService.getParameterValuesMap() );
+        Map<String, List<String>> parameterValuesMap = contextService.getParameterValuesMap();
+
+        if ( !parameterValuesMap.containsKey( "importReportMode" ) )
+        {
+            parameterValuesMap.put( "importReportMode", Collections.singletonList( "ERRORS_NOT_OWNER" ) );
+        }
+
+        MetadataImportParams params = importService.getParamsFromMap( parameterValuesMap );
 
         params.setUser( user )
+            .setImportReportMode( ImportReportMode.ERRORS_NOT_OWNER )
```

Comment:
I am confused by this. If the mode is set here to a constant why is it needed or useful to make sure it is set in `parameterValuesMap` ?

Reply:
Yes, that was a mistake, fixed now